includes/class-stats-dashboard.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * File containing the class Stats_Dashboard.
+ *
+ * @package wp-job-manager
+ */
+
+namespace WP_Job_Manager;
+
+/**
+ * Job listing stats for the jobs dashboard.
+ *
+ * @since $$next-version$$
+ */
+class Stats_Dashboard {
+
+	use Singleton;
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+
+		if ( ! Stats::is_enabled() ) {
+			return;
+		}
+
+		add_filter( 'job_manager_job_dashboard_columns', [ $this, 'add_stats_column' ] );
+		add_action( 'job_manager_job_dashboard_column_stats', [ $this, 'render_stats_column' ] );
+
+		add_filter( 'manage_edit-job_listing_columns', [ $this, 'add_stats_column' ], 20 );
+		add_action( 'manage_job_listing_posts_custom_column', [ $this, 'maybe_render_job_listing_posts_custom_column' ], 2 );
+	}
+
+	/**
+	 * Add a new column to the job dashboards.
+	 *
+	 * @param array $columns
+	 * @return array
+	 */
+	public function add_stats_column( $columns ) {
+		$columns['stats'] = __( 'Views', 'wp-job-manager' );
```

Comment:
WDYT about having a more specific name than `stats`? Just to avoid eventual possible conflicts with other plugins.

Maybe `wpjm_stats`?

Reply:
Also maybe we could move this to a private constant in this class?

Reply:
Moved it to a constant. I think we are fine with `stats`, this is just for our own post type and shortcode, the context is already WPJM there. The other stats plugin uses `total_views` so there is no conflict there.